Gems/PhysX/Code/Source/RigidBody.cpp

```diff
@@ -580,6 +580,21 @@ namespace PhysX
     {
         if (m_pxRigidActor)
         {
+            if (!isKinematic)
+            {
+                // check if any of the shapes on the rigid body would prevent switching to dynamic
+                if (!AZStd::all_of(m_shapes.begin(), m_shapes.end(), [](const AZStd::shared_ptr<PhysX::Shape>& shape) {
+                    return CanShapeComputeMassProperties(*shape->GetPxShape());}))
```

Comment:
uber nit: lots of stuff going on in that if condition, I suggest doing it in another line. Up to you.

````
const bool allShapesCanComputeMassProperties = 
    AZStd::all_of(m_shapes.begin(), m_shapes.end(), 
        [](const AZStd::shared_ptr<PhysX::Shape>& shape)
        {
            return CanShapeComputeMassProperties(*shape->GetPxShape());
        });
if (!allShapesCanComputeMassProperties)
....
````
````

Reply:
I'll double check but I think what I have is how clang formatting did it.